### PR TITLE
refactor: switch emails to @ipfs.tech

### PR DESCRIPTION
**Pull request by lidel**
> Context: https://github.com/protocol/bifrost-infra/issues/178#issuecomment-1195867284

This is a follow-up to https://github.com/ipfs/ipfs-website/pull/133 that switches `abuse@` and `security@` emails to `ipfs.tech` domain.

## TODO

- [x] confirm DNS is propagated, MX records for `ipfs.tech` exist,  and mailing lists responsible for `abuse@` and `security@` work

### CODE REVIEW

**Comment by BigLep** [comment]
@lidel : is this unblocked?

**Comment by lidel** [comment]
@BigLep afaik DNS configuration is done, but someone with gapps admin needs to add @ipfs.tech aliases.
We tracked  details in https://www.notion.so/pl-it-support/ipfs-io-ipfs-tech-c86465f384644541b907d19cfbc8918e but it seems someone nuked it? 

@andyschwab  do you know whats the latest on this?

**Comment by andyschwab** [comment]
@lidel - ipfs.tech should be fully functional for all active users and groups, including abuse@ and security@.

.io is still considered the "primary" domain in GW, but that should have no impact on email routing. We can change this too if necessary. I just tested changes and all looks good, but I encourage you to test as well and verify routing is functioning as you expect.

File: content/help.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ description: Learn where to get help and support in the extensive, diverse IPFS
...
 * **IPFS-related community announcements**: If you'd like to announce something to the IPFS community at large, use the [official IPFS discussion forums](https://discuss.ipfs.tech); we monitor the forums for things to include in the weekly IPFS newsletter! You can also inform people via [Matrix](https://docs.ipfs.tech/community/#chat) and the [other venues](#other-venues) listed below.
 * **Bugs and feature proposals**: IPFS-codebase-related bugs and feature proposals can be reported directly in their repositories (most are in the [IPFS](https://github.com/ipfs) and [IPFS Shipyard](https://github.com/ipfs-shipyard) GitHub orgs), for example [Kubo](https://github.com/ipfs/kubo) or [js-ipfs](https://github.com/ipfs/js-ipfs). If you're not sure where the best place to report a bug or request a feature is, ask in the [forums](https://discuss.ipfs.tech) and someone will point you in the right direction.
-* **Support for libp2p**: Questions specific to libp2p should use the [libp2p forums](https://discuss.libp2p.io) for faster results.
+* **Support for libp2p**: Questions specific to libp2p should use the [libp2p forums](https://discuss.libp2p.i
```
Comment (lidel) [inline content/help.md:11]:
```suggestion
* **Support for libp2p**: Questions specific to libp2p should use the [libp2p forums](https://discuss.libp2p.io) for faster results.
```

File: content/help.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ description: Learn where to get help and support in the extensive, diverse IPFS
...
 * **Reporting security issues**: To report [security issues](https://github.com/ipfs/community/blob/master/CONTRIBUTING.md#security-issues), please email <a href="mailto:security@ipfs.io">security@ipfs.io</a>.
 * **Reporting abuse**: To report [abuse](https://github.com/ipfs/community/blob/master/code-of-conduct.md), please email <a href="mailto:abuse@ipfs.io">abuse@ipfs.io</a>.
 * **Something else not listed here?** Ask in the [official IPFS discussion forums](https://discuss.ipfs.tech).
```
Comment (lidel) [inline content/help.md:14]:
```suggestion
* **Reporting security issues**: To report [security issues](https://github.com/ipfs/community/blob/master/CONTRIBUTING.md#security-issues), please email <a href="mailto:security@ipfs.tech">security@ipfs.tech</a>.
* **Reporting abuse**: To report [abuse](https://github.com/ipfs/community/blob/master/code-of-conduct.md), please email <a href="mailto:abuse@ipfs.tech">abuse@ipfs.tech</a>.
* **Something else not listed here?** Ask in the [official IPFS discussion forums](https://discuss.ipfs.tech).
```

**Pull request merged**